code/game/gamemodes/shadowling/shadowling_abilities.dm

```diff
@@ -61,7 +61,7 @@
 	clothes_req = 0
 	range = 5
 	action_icon_state = "veil"
-	var/blacklisted_lights = list(/obj/item/device/flashlight/flare, /obj/item/device/flashlight/slime)
+	var/blacklisted_lights = list(/obj/item/device/flashlight/slime)//Slightly buffed, requiring slimes instead of slimes and flares
```

Comment:
> slightly buffed

I can tell you I've never seen any defence against shadowlings that didn't involve botany spamming torches. Taking out botany is one of the key points to a shadowling victory.

I'm all for Xeno slimelights getting more love, but DAMN son. At least give people the ability to build a bonfire out of broken wooden chairs to huddle around.